### DISCUSSION TITLE
[Feature] Add the upload button from the web app to the mobile app

### DISCUSSION THREAD

**Discussion by leftbones** [immich-app/immich#8225] [Feature Request]
### The feature

On the web app, there is a button in the upper right corner to upload directly from the device's filesystem. On the mobile app, there is no such button.

As far as I can tell, the only way to directly upload files using the mobile app is to add them to an album that I have selected as a backup, manually select and upload the photos, then delete them from my phone.

I don't want to have any albums on my phone connected to the app, period. I only want to view the files I've uploaded, which is why the above method is even less desirable.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#7075
